pkg/scheduler/framework/plugins/volumezone/volume_zone_test.go

```diff
@@ -317,16 +317,18 @@ func TestWithBinding(t *testing.T) {
 			Node: testNode,
 		},
 		{
-			name:       "unbound volume empty storage class",
-			Pod:        createPodWithVolume("pod_1", "vol_1", "PVC_EmptySC"),
-			Node:       testNode,
-			wantStatus: framework.NewStatus(framework.Error, "PersistentVolumeClaim was not found: \"PVC_EmptySC\""),
+			name: "unbound volume empty storage class",
+			Pod:  createPodWithVolume("pod_1", "vol_1", "PVC_EmptySC"),
+			Node: testNode,
+			wantStatus: framework.NewStatus(framework.Error,
```

Comment:
this format seems wrong, please revert

Reply:
@alculquicondor   I have `go fmt` the codes,  it is not wrong.

see line334 and 340 also have the similar format

Reply:
oh I see, you put a new line